src/platform/user/authentication/utilities.js

```diff
@@ -20,13 +20,14 @@ function sessionTypeUrl(
   version = 'v0',
   application = null,
   to = null,
+  queryParams = null,
```

Comment:
Is there a reason why you have this default set to null instead of an empty object literal?